query/src/main/kotlin/com/linecorp/kotlinjdsl/query/CriteriaUpdateQuerySpec.kt

```diff
@@ -0,0 +1,18 @@
+package com.linecorp.kotlinjdsl.query
+
+import com.linecorp.kotlinjdsl.query.clause.from.FromClause
+import com.linecorp.kotlinjdsl.query.clause.from.JoinClause
+import com.linecorp.kotlinjdsl.query.clause.hint.JpaQueryHintClause
+import com.linecorp.kotlinjdsl.query.clause.hint.SqlQueryHintClause
+import com.linecorp.kotlinjdsl.query.clause.where.CriteriaQueryWhereClause
+import com.linecorp.kotlinjdsl.query.spec.expression.ColumnSpec
+
+interface CriteriaUpdateQuerySpec<T> {
+    val targetEntity: Class<T>
+    val from: FromClause
+    val join: JoinClause
+    val where: CriteriaQueryWhereClause
+    val jpaHint: JpaQueryHintClause
+    val sqlHint: SqlQueryHintClause
+    val params: Map<ColumnSpec<*>, Any?>
```

Comment:
-- english
When I read the code again, params feel ambiguous because from, join, and where are also params.

How about defining SetClause?

-- korean
코드를 다시 읽어보았을 때 from, join, where 또한 param이기 때문에 params는 애매하게 느껴집니다.

SetClause를 정의하는 것은 어떤가요?

Reply:
-- english
It would be a good idea to create SetClause so that the QueryCreator applies SetClause through the SetClause.apply method.

-- korean
SetClause를 만들어서 SetClause.apply 메소드를 통해 QueryCreator가 SetClause를 적용하게 하는 것이 좋을 것 같습니다.

Reply:
I have defined SetClause
-- korean
SetClause 정의 했습니다.